runtime/gc_structs/ConstantPoolClassSlotIterator.cpp

```diff
@@ -61,7 +61,10 @@ GC_ConstantPoolClassSlotIterator::nextSlot()
 
```

Comment:
you could bring down declaration of slotType and slotPtr, to the point they are initialized for the first time.

Reply:
I see 3 missing spaces between while/if and (', in this method